cookbooks/aws-parallelcluster-install/libraries/helpers.rb

```diff
@@ -90,3 +90,13 @@ def append_if_not_present_grub_cmdline(attributes, grub_variable)
     end
   end
 end
+
+def efa_installed?
+  dir_exist = ::Dir.exist?('/opt/amazon/efa')
+  if dir_exist
+    modinfo_efa_stdout = Mixlib::ShellOut.new("modinfo efa").run_command.stdout
+    efa_installed_packages_file = Mixlib::ShellOut.new("cat /opt/amazon/efa_installed_packages").run_command.stdout
+    Chef::Log.info("`/opt/amazon/efa` directory already exists. \nmodinfo efa stdout: \n#{modinfo_efa_stdout} \nefa_installed_packages_file_content: \n#{efa_installed_packages_file}")
+  end
+  dir_exist
+end
```

Comment:
Thought: I just wonder if this shouldn't go to networking library in common cookbook.

Reply:
Moved, good point!